features/FxA-53-send-sms/README.md

```diff
@@ -0,0 +1,444 @@
+
+3. Deep link the user into the app and pre-populate the email field.
+
+4. Pass a session token or code in the deeplink and avoid any login whatsoever (with a confirmation prompt on originating device)
+
+## Outcomes
+
+The outcome we are looking for is a slick user flow that makes it obvious to users that in order to complete their setup of Sync, they needs to add another device after email confirmation.
+
+## Hypothesis
+
+If we offer to download via SMS to remove friction around setting up a second device for firstrun users on desktop, then we will see an increase in multi-device users within the first 7 days of registration because we have observed that less than [30% of registration](https://next.adjust.com/#/statistics/overview/default/gqrwnbypw28p/hqnqs2?from=2016-10-01&to=2016-10-31&range=last-month&sort=installs&reverse=true) (65k clicks / 210k registration / month) engage with our mobile download buttons and [at best 26%](https://sql.telemetry.mozilla.org/queries/1254#2173) (8% 7d md / 30% ctr) of them follow through to an install within 7 days after clicking.
+
+## Metrics
+
+We will know our hypothesis to be true when we see the proportion of multi-device users increase after 48h and 7d of registration to FxA+Sync on firstrun.
+
+<img src="./image_0.png" / alt="New signups that connect another device" />
+
+From: https://sql.telemetry.mozilla.org/queries/1254#2173
+
+We will also want to keep an eye out for:
+
+* **Email confirmation rate**: we should not impact this negatively in any way.
+
+    * Measured with telemetry and visualized in re:dash
+
+* **Volume of SMS sent**: Will confirm if users are adopting new mechanism.
+
+    * Measured with SMS provider
+
+* **SMS send rate**: Will confirm if users are adopting new mechanism.
+
+* **Click rate**: Will allow to see if messages are delivered and users are clicking on them.
+
+* **Install rate from SMS**: Will allow to see if users are successfully installing after install click on link in SMS
+
+* **Install rate** from confirmation page on mobile: If users confirmed their email on mobile, we will drive users to install via smart banner. (install rate)
+
+### Flow events
+
+<table>
+  <tr>
+    <td>flow.sms.view</td>
+    <td># of people that see the send_sms screen</td>
+  </tr>
+  <tr>
+    <td>flow.sms.engage</td>
+    <td># of people that interact with form.</td>
+  </tr>
+  <tr>
+    <td>flow.sms.submit</td>
+    <td># of people that try to send a message.</td>
+  </tr>
+  <tr>
+    <td>sms.1.sent</td>
+    <td># of people that we successfully send a message to.</td>
+  </tr>
+  <tr>
+    <td>flow.sms.link.learn_more</td>
+    <td># of people that click on "learn more"</td>
+  </tr>
+  <tr>
+    <td>flow.sms.link.why</td>
+    <td># of people that click on “why is this required”</td>
+  </tr>
+  <tr>
+    <td>flow.sms.link.maybe_later</td>
+    <td># of people that click “maybe later”</td>
+  </tr>
+</table>
+
```

Comment:
I apparently added a table at this spot after you moved the doc to Github. See: https://docs.google.com/document/d/133_Cu0TlWQZIw-ihMH1s-U_80I12MXZcvHQEuFhWbPg/edit#heading=h.eeevjsod1m6i

I just want to outline how we will track the SMS steps (clicks + installs).